.github/workflows/nf-core-linting.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
         # FIXME: Remove this when nf-core modules lint stabilizes and install stable release
       - name: Install nf-core tools development version
-        run: python -m pip install --upgrade --force-reinstall git+https://github.com/nf-core/tools.git@dev
+        run: python -m pip install --upgrade --force-reinstall git+https://github.com/nf-core/tools.git@subworkflow_hacks
```

Comment:
Not "_hacks" if it works, right? 😉

Reply:
Lol well it was ignoring them because we need to update the standards to match, but there's already subworkflows flowing in, so we need to punt on that. https://github.com/nf-core/tools/pull/1319

I'm trying to fix the tests first, and then I'll be able to test for all of these new things.